sops/alerts/pod_crash_looping.adoc

```diff
@@ -0,0 +1,24 @@
+:toc:
+:toc-placement!:
+
+= ThreescalePodCrashLooping
+
+toc::[]
+
+== Description
+
+* This alert will trigger if a pod is on `CrashLoop` state (being restarted regularly)
+
+== Troubleshooting
+
+* Check pod logs/events to see the reason of the `CrashLoop`
+* Check if maybe container image registry is not reachable
```

Comment:
Are you sure that this one may trigger a crash loopback? Or the only the ThreescaleContainerWaiting alert?